src/utils/axiosWithAuth.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const axiosWithAuth = () => {
+  const token = localStorage.getItem('okta-token-storage');
+  const parsedToken = JSON.parse(token);
+
+  return axios.create({
+    baseURL: process.env.REACT_APP_API_URI,
+    headers: {
+      Authorization: 'Bearer ' + parsedToken.idToken.value,
+    },
+  });
+};
+
+export default axiosWithAuth;
```

Comment:
We decided during yesterday's standup that we would be using Paul's axiosWithAuth which is already merged, so we can probably just remove this.